src/data/api.js

```diff
@@ -70,25 +65,27 @@ export async function fetchLatestAttempt(courseId) {
 
 export async function pollExamAttempt(pollUrl, sequenceId) {
   let data;
+
+  // sites configured with only edx-proctoring must have pollUrl set
   if (pollUrl) {
     const edxProctoringURL = new URL(
       `${getConfig().LMS_BASE_URL}${pollUrl}`,
     );
     const urlResponse = await getAuthenticatedHttpClient().get(edxProctoringURL.href);
     data = urlResponse.data;
-  } else if (sequenceId && getConfig().EXAMS_BASE_URL) {
-    data = await fetchLatestExamAttempt(sequenceId);
 
+  // sites configured with edx-exams expect sequenceId if pollUrl is not set
+  // and the learner is viewing the exam sequence
+  // (the exam sequence is not consumed outside the exam sequence, e.g. in the course view)
```

Comment:
I think this comment could be a bit clearer, specifically the piece in parenthesis.